src/pybind/cephfs/cephfs.pyx

```diff
@@ -150,7 +148,10 @@ cdef extern from "cephfs/libcephfs.h" nogil:
 
 
 class Error(Exception):
-    pass
+    def __init__(self, errno, strerror):
+        super().__init__(errno, strerror)
```

Comment:
I was surprised to see super with no arguments work - cython transforms it so it works with python2